conf/st2.conf.sample

```diff
@@ -6,17 +6,21 @@
 enable = True
 
```

Comment:
Well, first I also need to fully automate this one.

This means adding a step to Makefile so we don't forget to update it like we did many times now :P